### PR TITLE
add separate directories for models, activities, and fragments

### PR DESCRIPTION
**Pull request by styliii**
Probably should have put in the pull request earlier.  Let me know if this isn't mergeable for you guys.

### CODE REVIEW

File: WealthNOW/app/src/main/java/com/finnerds/wealthnow/WealthNow/activities/MainActivity.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.finnerds.wealthnow.activites;
+package com.finnerds.wealthnow.WealthNow.activities;
```
Comment (mikeng13) [inline WealthNOW/app/src/main/java/com/finnerds/wealthnow/WealthNow/activities/MainActivity.java]: Why are we adding an extra wealthnow subdirectory? 

I think we are fine with com.finnerds.wealthnow.activites

Comment (styliii) [inline WealthNOW/app/src/main/java/com/finnerds/wealthnow/WealthNow/activities/MainActivity.java]: I don't think it merges correctly. I wanted to have an activities, models
and fragments directory.

Comment (mikeng13) [inline WealthNOW/app/src/main/java/com/finnerds/wealthnow/WealthNow/activities/MainActivity.java]: Lol. I'm worried it'll merge what the diff is showing. Can you try pushing another remote branch with the correct changes?

**Comment by styliii** [comment]
ok i'll close this, and we can merge in @mikeng13's branch into ours @enginnerd

**Pull request closed without merging**